package.json

```diff
@@ -30,7 +30,7 @@
   },
   "homepage": "https://github.com/asyncapi/avro-schema-parser#readme",
   "devDependencies": {
-    "@asyncapi/parser": "^1.14.1",
+    "@asyncapi/parser": "^1.15.0",
```

Comment:
It's a dev dependency, so I don't think so that PR is needed. If you only use this dependency for testing then the new changes (in parser core) do not have any breaking changes, so just bumping up the dependencies and the package does not make sense.

Reply:
yes you have absolutely right :) it appears that my holidays was the best effect ^^

Reply:
hah no problem, glad you got some rest. Generally this dependency will be bumped automatically, but without releasing the package - our workflow will take care of it :)